vim-patch:8.0.0558,8.0.0562

janlazo opened the pull request:
**vim-patch:8.0.0558: :ownsyntax is not tested**

Problem:    The :ownsyntax command is not tested.
Solution:   Add a test. (Dominique Pelle, closes vim/vim#1622)
vim/vim@f8ec998

**vim-patch:8.0.0562: not enough test coverage for syntax commands**

Problem:    Not enough test coverage for syntax commands.
Solution:   Add a few more tests. (Dominique Pelle, closes vim/vim#1624)
vim/vim@ea58815

janlazo committed:
vim-patch:8.0.0558: :ownsyntax is not tested

Problem:    The :ownsyntax command is not tested.
Solution:   Add a test. (Dominique Pelle, closes vim/vim#1622)
https://github.com/vim/vim/commit/f8ec998613d8037e345f4e7e08460dfc15c528a9

justinmk commented:
can you also include the whole `Test_syntax_invalid_arg()` test? Looks like we missed that somehow in 20e0cb8d47e4.

janlazo committed:
oldtests: add conceal check for patch 8.0.0562

janlazo committed:
vim-patch:8.0.0562: not enough test coverage for syntax commands

Problem:    Not enough test coverage for syntax commands.
Solution:   Add a few more tests. (Dominique Pelle, closes vim/vim#1624)
https://github.com/vim/vim/commit/ea588154d00aec288e48b344b2685e46bea99cd6

justinmk commented:
Thanks! In the merge commit I moved things around (to match Vim source) so that hopefully `patch` works better next time.
